### PR TITLE
[TH2-5045] Migrate to th2 transport and versions update

### PR DESCRIPTION
**Pull request by OptimumCode**

### CODE REVIEW

File: .github/workflows/dev-docker-matrix-publish.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
           - conn-fix
           - conn-ntg
           - conn-soup
-    uses: th2-net/.github/.github/workflows/matrix-java-docker-dev.yml@main
+    uses: th2-net/.github/.github/workflows/matrix-java-docker-dev.yml@matrix-java
```
Comment (Nikita-Smirnov-Exactpro) [inline .github/workflows/dev-docker-matrix-publish.yml]: Use main branch after test

File: .github/workflows/dev-release-docker-matrix-publish.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        project:
+          - conn-fix
+          - conn-ntg
+          - conn-soup
+    uses: th2-net/.github/.github/workflows/matrix-java-docker.yml@matrix-java
```
Comment (Nikita-Smirnov-Exactpro) [inline .github/workflows/dev-release-docker-matrix-publish.yml]: Use main branch after test

File: .github/workflows/docker-matrix-publish.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ jobs:
           - conn-fix
           - conn-ntg
           - conn-soup
-    uses: th2-net/.github/.github/workflows/matrix-java-docker.yml@main
+    uses: th2-net/.github/.github/workflows/matrix-java-docker.yml@matrix-java
```
Comment (Nikita-Smirnov-Exactpro) [inline .github/workflows/docker-matrix-publish.yml]: Use main branch after test

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]

File: .github/workflows/dev-release-docker-matrix-publish.yml (hunk 2 above)
Comment (OptimumCode) [reply .github/workflows/dev-release-docker-matrix-publish.yml]: I will once it is merged in the main branch in .github repo

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**